fem/restriction.cpp

```diff
@@ -653,7 +653,8 @@ ConformingFaceRestriction::ConformingFaceRestriction(
    : ConformingFaceRestriction(fes, f_ordering, type, true)
 { }
 
-void ConformingFaceRestriction::Mult(const Vector& x, Vector& y) const
+void ConformingFaceRestriction::MultInternal(const Vector& x, Vector& y,
+                                             const bool useAbs) const
```

Comment:
One option here (probably not too important) is to make `useAbs` a template parameter, than will avoid the runtime check at every iteration. Likely makes little to no difference though…